app/Models/Foundation/Summit/ProposedSchedule/SummitProposedSchedule.php

```diff
@@ -197,6 +197,32 @@ public function getScheduledSummitEventsByLocationAndDateRange(
         return $this->scheduled_summit_events->matching($criteria)->toArray();
     }
 
+    /**
+     * @param \DateTime $date
+     * @param SummitAbstractLocation|null $location
+     * @return SummitProposedScheduleSummitEvent[]
+     */
+    public function getProposedPublishedEventsBeforeThan(\DateTime $date, SummitAbstractLocation $location):array {
+
+        $criteria = Criteria::create();
+        $criteria->where(Criteria::expr()->lt('end_date', $date));
+        $criteria->andWhere(Criteria::expr()->eq('location', $location));
+        return $this->scheduled_summit_events->matching($criteria)->toArray();
```

Comment:
dont use array bc its hydrates all the collection

Reply:
done